src/js/UserUI.js

```diff
@@ -0,0 +1,24 @@
+export default class UserUI {
+  constructor() {
+    this.AuthHead = document.querySelector('.auth');
+    this.UserHead = document.querySelector('.user-info');
+    this.userName = document.querySelector('#username-in-head');
+
+  }
+
+  init() {
+    const {AuthHead, UserHead, userName} = this;
+    const user = localStorage.getItem('username')
```

Comment:
we have User class when u can get username from localStorage `User.username`
and i think better check if we have token `User.token`

Reply:
Exactly! Done)